Rakefile

```diff
@@ -9,4 +9,12 @@ RSpec::Core::RakeTask.new
 require 'yard'
 YARD::Rake::YardocTask.new
 
+# Rollbar
+require 'rollbar/rake_tasks'
+task :environment do
+  Rollbar.configure do |config|
+    config.access_token = '4d1ae93436594677844a781e7812faed'
```

Comment:
I've deleted and generated new tokens on Rollbar.

This change is good but let's not commit tokens anymore to the repositories. It's a bad practice and we (not just @MirkoMignini - we are all responsible) shouldn't do it.